chain/client/tests/catching_up.rs

```diff
@@ -641,4 +642,209 @@ mod tests {
         })
         .unwrap();
     }
+
+    enum OrphansPhases {
+        WaitingForSendingTx,
+        HoldingBlock,
+        ValidatingOrphans,
+    }
+
+    #[test]
+    fn test_catchup_orphans_under_epoch() {
+        test_catchup_orphans_common(2, 5)
+    }
+
+    #[test]
+    fn test_catchup_orphans_over_epoch() {
+        test_catchup_orphans_common(2, 10)
+    }
+
+    fn test_catchup_orphans_common(height_hold: u64, height_release: u64) {
+        if !cfg!(feature = "expensive_tests") {
```

Comment:
Use the proper `[cfg(feature="expensive_test")]` before the test.
We changed the CI to compile with all features, so no need for this cluchery :)

Reply:
#[cfg(feature = "expensive_test")] doesn't work. Let's discuss offline.